paywall/src/__tests__/hooks/usePaywallConfig.test.js

```diff
@@ -93,6 +93,7 @@ describe('usePaywallConfig hook', () => {
 
     const myConfig = {
       ...defaultValue,
+      type: 'paywall',
```

Comment:
this simply verifies that the `type` field properly validates and is passed to the component using the hook. Because of the way the test is designed, this is the only change necessary. Prior to adding the validation, this 1-line change would have made the test fail